content/workshops/security_containers/exercise1.0.adoc

```diff
@@ -54,7 +55,7 @@ image::login.png[Login to Cockpit]
 ----
 Username: cockpit
 
-Password: DC17!
+Password: redhat!@#
```

Comment:
This is configurable from the playbooks, so I probably edit that to say, ask your instructor for password, or something like that.